Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang:1.11.5 AS builder
-ADD . /src
-RUN cd /src \
- && go test . \
- && go build -o binary
+FROM golang:alpine as builder
+RUN apk add git && \
+	go get -d -v github.com/terorie/od-database-crawler
+WORKDIR /go/src/github.com/terorie/od-database-crawler
+RUN	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /go/od-database-crawler .
 
-FROM alpine
-WORKDIR /app
-COPY --from=builder /src/binary /app/
-ENTRYPOINT [ "/app/binary", "server" ]
+FROM scratch
+COPY script/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /go/od-database-crawler /go/
+VOLUME [ "/go" ]
```

Comment:
Not sure about this, but this will add the executable itself to the volume, right?

Reply:
program generates the queue, logs and other files in the same path location as the program, if the queue is stored inside the container and the container gets rebuild then the queue goes which could cause issues.

Reply:
I agree having a volume is good.
What I mean though is, can you move the `od-database-crawler` executable itself to `/bin` so it won't be added to the volume?

Reply:
if you add a flag so you can set where the files are generated sure. however if i do that and don't set the volume to /bin then its going to generate all the files and folders in the bin folder and be inside the image causing issues with the queue if the container is re-deployed

Reply:
The files are generated in the current working directory, which is changeable (With `cd`)